packages/apps/plugins/plugin-table/src/components/ObjectTable/ObjectTable.tsx

```diff
@@ -115,7 +110,7 @@ export const ObjectTable: FC<ObjectTableProps> = ({ table, role, stickyHeader, g
     });
 
     return builder.createColumns();
-  }, [space, tables, reactDeps(table, table.schema), newObject]);
+  }, [space, tables, table, table.schema, newObject]);
```

Comment:
I'm not super familiar with this plugin, but this looks like a step in the right direction to me 👍